app/src/main/java/io/zenandroid/onlinego/views/BoardView.kt

```diff
@@ -112,9 +125,13 @@ class BoardView : View {
         }
     }
 
-    private fun drawTextCentred(canvas: Canvas, paint: Paint, text: String, cx: Float, cy: Float) {
+    private fun drawTextCentred(canvas: Canvas, paint: Paint, text: String, cx: Float, cy: Float, ignoreAscentDescent: Boolean = false) {
```

Comment:
can you please explain this `ignoreAscentDescent`?

Reply:
Letters with descent (like the Q) would have a different vertical alignment from the others otherwise; this parameter (disabled by default) allows to center text ignoring the descent. Without it the Q would be displaced (centered higher than the other letters) on the screenshots in the comment below

Reply:
for now yes, but the method is useful when we want a letter with descent to be "perfectly centered" (for example inside a cell, like the move numbers) instead of "aligned as if writing in lined paper"

Reply:
ok, let's keep it then...